cmd/cmd_controller.js

```diff
@@ -291,9 +291,11 @@ class EmbarkController {
         engine.startService("namingSystem");
         engine.startService("console");
         engine.startService("pluginCommand");
-        callback();
+        engine.events.on('check:backOnline:Ethereum', function () {
+          callback();
+        });
```

Comment:
Also, can you elaborate on how this change is related to the fix you're proposing?

Reply:
without that there is a race condition, the ens contract are not deployed when using the console command without that.
This is due to the fact the ENS contracts are setup when we are connected to web3.
So before deploying we make sure the node is connected.
This is a pattern use for other command.